content/webapp/pages/events/[eventId]/index.tsx

```diff
@@ -294,7 +306,7 @@ const EventPage: NextPage<EventProps> = ({
             <EventbriteButtons event={event} />
             {event.thirdPartyBooking && (
               <>
-                {event.isCompletelySoldOut ? ( // TODO online sold out / versus normal sold out
```

Comment:
Please lmk if these are TODOs we would like to address, but as they are 3 years old I chose to remove.